### PR TITLE
feat: Add time zone fields to events

### PR DESCRIPTION
**Pull request by peterstarling**
Adds support for time-zones for start and end dates in the Event entity

### CODE REVIEW

File: packages/squidex/schema/schemas/events.json [hunk 1]
```diff
@@ -60,6 +60,23 @@
           "isRequired": false
         }
       },
+      {
+        "name": "startDateTimeZone",
+        "isHidden": false,
+        "isLocked": false,
+        "isDisabled": false,
```
Comment (johnytiago) [inline packages/squidex/schema/schemas/events.json:67]:
```suggestion
        "isDisabled": true,
```

File: packages/squidex/schema/schemas/events.json [hunk 2]
```diff
@@ -75,6 +92,23 @@
           "isRequired": false
         }
       },
+      {
+        "name": "endDateTimeZone",
+        "isHidden": false,
+        "isLocked": false,
+        "isDisabled": false,
```
Comment (johnytiago) [inline packages/squidex/schema/schemas/events.json:99]:
```suggestion
        "isDisabled": true,
```

**Comment by johnytiago** [comment]
The date timezones shouldn't be editable through the UI

File: packages/model/src/event.ts [hunk 3]
```diff
@@ -8,7 +8,9 @@ export interface EventResponse {
   id: string;
   title: string;
   startDate: string;
+  startDateTimeZone: string;
```
Comment (johnytiago) [inline packages/model/src/event.ts:11]: I thought the TZs wouldn't propagate directly to the FE

Comment (jeysal) [reply packages/model/src/event.ts:11]: with the new approach suggested by @peterstarling they do, this would be something like `"Europe/Lisbon"` afaik

**Review by johnytiago — APPROVED** [review]

**Pull request merged**